custom_components/ha_washdata/playground.py

```
@@ -200,6 +207,103 @@ def _coerce_bool(value: Any) -> bool:
+    for opt_key, (cfg_key, coerce) in _MATCH_OVERRIDE_KEYS.items():
+        value = cfg.get(cfg_key, MATCH_DEFAULTS_BY_OPTION.get(opt_key))
+        if value is None:
+            continue
+        try:
+            out[opt_key] = coerce(value)
+        except (TypeError, ValueError):  # pragma: no cover - defensive
+            continue
+    return out
+
+
+def sanitize_setting_values(values: Any) -> dict[str, Any]:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `values`

(ANN401)